### ISSUE TITLE
Cache sometimes returns an object

### ISSUE THREAD

**Issue by r-stein** [SublimeText/LaTeXTools#1081]
The cache sometimes returns an object instead of the cached entry.

E.g. for bib files:
```
Bib files found: 
<object object at 0x00000000025B8990>
Traceback (most recent call last):
....
completions = run_plugin_command('get_entries', *bib_files)
TypeError: run_plugin_command() argument after * must be a sequence, not object
```

This also occured for completions (array) and other cached files.

C-l, C-backspace or C-l, C-d, C-c temporary solves the error and the cache is working normal afterwards.

**Comment by ig0774** [comment]
I never quite tracked down the issue, but applying #1058 (and probably deleting existing caches) fixed this for me. I haven't had time to test #1058 on Linux, but otherwise it's been working for me.

**Comment by ig0774** [comment]
This is fixed in v3.15.0.
